From: SeongJae Park <sj@kernel.org>
Date: Sat, 23 Nov 2024 07:35:34 -0800
Subject: [PATCH] mm/damon/core: fix a bad line on comment

A line in the middle of comment is not having prefix ' * '.  Add it.

Fixes: d9418cac100e2 ("mm/damon/core: implement damon_call()")
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202411231318.5NR1XkPk-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index 44e0ab6362b3..a0384c8ab308 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -1205,7 +1205,7 @@ static bool damon_is_running(struct damon_ctx *ctx)
  * internal data of &struct damon_ctx including &struct damon_region objects
  * without additional locking.  The return code of the callback function will
  * be saved in &->return_code of @control.
-
+ *
  * If DAMON is deactivated by watermarks or terminated before the function is
  * called back, the request is canceled.
  *
-- 
2.39.5

